src/figuresClasses.ts

```diff
@@ -1,19 +1,99 @@
-export interface Figure {
+enum Shape {
+  triangle = 'triangle',
+  circle = 'circle',
+  rectangle = 'rectangle',
+}
+
+enum Color {
+  red = 'red',
+  green = 'green',
+  blue = 'blue',
+}
 
+export interface Figure {
+  color: Color;
+  shape: Shape;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
```

Comment:
you have the Shape enum. use it here instead of hardcoded values
```suggestion
  shape: Shape = Shape.triangle;
```